.github/workflows/ollama_workflow.yaml

```diff
@@ -0,0 +1,48 @@
+name: CI for Ollama Notebooks
```

Comment:
Note that the objective now is to run all (code) notebooks from Replicate by default. So Ollama testing should be confined to the "Kitchen", and maybe some repos where we don't have the necessary models on Replicate.

Reply:
cool, once those get moved we can drop or move this workflow

Reply:
Agreed with the point.  I'll write a ticket for this objective.